docs/en/developer/create-web-component/VUE.md

```diff
@@ -341,61 +396,109 @@ To add Chrome (the standard border around portlets):
 
 To remove Chrome:
 
-```xml
+``` xml
     <parameter>
         <name>chromeStyle</name>
         <value>no-chrome</value>
     </parameter>
 ```
 
+To grant permission to everyone to browse for the web component and select it:
+
+``` xml
+    <group>Everyone</group>
+    <permissions>
+        <permission system="UP_PORTLET_SUBSCRIBE" activity="BROWSE">
+            <group>Everyone</group>
+        </permission>
+    </permissions>
+
+```
+
 ### Add webjar to resource server
 
-In the `overlays/resource-server/build.gradle` file in the uPortal-start
+In the **overlays/resource-server/build.gradle** file in the uPortal-start
 project, add the following runtime dependency:
 
-```gradle
+```
     runtime "org.webjars.npm:uportal__{component-name}:{version}@jar"
 ```
 
 For example:
 
-```gradle
+```
     runtime "org.webjars.npm:uportal__weather-thingy:0.1.0-SNAPSHOT@jar"
 ```
 
-## Appendix
+### Rebuild uPortal-start
 
-### Node.js installation
+Rebuild the **uPortal-start** project to populate the database with the new
+portlet definition and load the new webjar into the resource server.
 
-#### Mac OS X
+```
+./gradlew portalInit
+```
 
-1. [MacPorts](https://www.macports.org/)
-2. [HomeBrew](https://brew.sh/)
-3. macOS installer (.pkg) from [Node.js website](https://nodejs.org/)
+When you start uPortal, you should be able to find the new component when
+you select *Customize > Add Stuff*.
 
-##### With MacPorts
+# Appendix
```

Comment:
🎨 ♿️ watch out 🙂  "Appendix" is necessarily a child of `# Develop Vue Web Components for uPortal` in the web page.
so suggest 
```
# Develop Vue Web Components for uPortal
....
## Appendix

```

Reply:
```suggestion
## Appendix
```

Reply:
Done.